### ISSUE TITLE
[Block Domain] Fake Crypto mixer websites list Phishing

### ISSUE THREAD

**Issue by Jackofusu80** [MetaMask/eth-phishing-detect#13797]
**Domain:** kryptomixer.io , cryptomixer2.com , cryptomixer.hk , crypto-mixer.cc , cryptomixer.biz , cryptomixer.si , cryptamixer.io , crypmix4m5iunofa25mpmiihdb56oaqg57tvrebqatc6otn3w65qhlid.com , coinmixerbtc.com , smartmixers.net , anonymix-btc.com , smartmixio.com , blander.io , bllndmixer.com , chipmixer-com.com

**Details:**
Phishing websites that mimic legitimate bitcoin mixer and crypto mixer sites are on the rise, putting unsuspecting users at risk of losing their hard-earned crypto investments. Some of these phishing websites, such as bllndmixer.com, are already offering Ethereum, which is a significant risk for MetaMask users. 
The danger also exists with all of the phishing domains mentioned above because they use general crypto (crypto, coin & chips) words on their websites or domain names, so there is also a danger for Metamask users from these phishing domains.

Other phishing websites that offer different cryptocurrencies and Bitcoin can also add Ethereum soon, as their domain names often involve terms like "Crypto", "Coin", or "Chip". This puts MetaMask and Ethereum users at significant risk as some scam phishing websites have already done this. 
It is especially concerning that this phishing website "smartmixers.net" already has announced on their main page that they will be adding Ethereum soon, which puts MetaMask and Ethereum users at even greater risk. We must remain vigilant and continue to report these phishing websites to protect innocent users from falling victim to their fraudulent activity.

It is crucial to add all these phishing domains to the MetaMask phishing detector system to protect MetaMask and crypto users from these scammers. 

Scammers behind these websites often trick users with slick-looking websites that mimic legitimate ones, complete with similar logos, designs, and even fake user reviews. 

In conclusion, it is important for MetaMask to add these phishing websites to the MetaMask Phishing detector system to protect crypto users and MetaMask users. 

- [X] I have checked to make sure that [there is not a duplicate issue](https://github.com/MetaMask/eth-phishing-detect/issues)

**Comment by AlexHerman1** [comment]
this was added, thanks for the sub!
